autohooks/plugins/pylint/pylint.py

```diff
@@ -83,16 +83,21 @@ def precommit(config=None, **kwargs):  # pylint: disable=unused-argument
     arguments = get_pylint_arguments(config)
 
     with stash_unstaged_changes(files):
-        args = ['pylint']
-        args.extend(arguments)
-        args.extend([str(f.absolute_path()) for f in files])
-
-        status = subprocess.call(args)
-        str_files = ', '.join([str(f.path) for f in files])
-
-        if status:
-            fail('Linting error(s) found in {}.'.format(str_files))
-        else:
-            ok('Linting {} was successful.'.format(str_files))
-
-        return status
+        for f in files:
+            cmd = ['pylint']
+            cmd.extend(arguments)
+            cmd.append(str(f.absolute_path()))
+            proc = subprocess.Popen(
+                cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE
+            )
+            out_, _ = proc.communicate()
+            if out_:
+                out_ = out_.decode(encoding='utf-8').split('\n')
```

Comment:
Maybe we should handle other encodings more carefully?

See https://docs.python.org/3/library/codecs.html#error-handlers for possible options.

```suggestion
                out_ = out_.decode(encoding='utf-8', errors='replace').split('\n')
```

Reply:
Another question are the pylint errors put into stdout or stderror?

Reply:
stdout. :)
It is no "Error", but an output ...